be/src/olap/rowset/segment_v2/page_io.cpp

```diff
@@ -115,7 +115,10 @@ Status PageIO::read_and_decompress_page(const PageReadOptions& opts, PageHandle*
                                         Slice* body, PageFooterPB* footer) {
     opts.sanity_check();
     opts.stats->total_pages_num++;
+     Status st = Status::Error(ErrorCode::INTERNAL_ERROR,"in page_io now, opts.use_page_cache:{}", opts.use_page_cache);
```

Comment:
This is debug code, do not submit to master

Reply:
Already modified